docs/integrations/engines/databricks.md

```diff
@@ -8,19 +159,32 @@
 pip install "sqlmesh[databricks]"
 ```
 
-### Connection info
+### Connection options
 
-If you are always running SQLMesh commands directly on a Databricks Cluster (like in a Databricks Notebook using the [notebook magic commands](../../reference/notebook.md)) then the only relevant configuration is `catalog` and it is optional.
-The SparkSession provided by Databricks will be used to execute all SQLMesh commands.
+Databricks provides multiple computing options and connection methods. This section explains how to use them with SQLMesh.
+
+#### Databricks SQL Connector
+
+SQLMesh uses the [Databricks SQL Connector](https://docs.databricks.com/dev-tools/python-sql-connector.html) to connect to Databricks by default.
+
+#### Databricks Connect
 
-Otherwise SQLMesh's Databricks implementation uses the [Databricks SQL Connector](https://docs.databricks.com/dev-tools/python-sql-connector.html) to connect to Databricks by default.
-If your project contains PySpark DataFrames in Python models then it will use [Databricks Connect](https://docs.databricks.com/dev-tools/databricks-connect.html) to connect to Databricks.
-SQLMesh's Databricks Connect implementation supports Databricks Runtime 13.0 or higher. If SQLMesh detects you have Databricks Connect installed then it will use it for all Python models (so both Pandas and PySpark DataFrames).
+If your project uses PySpark DataFrames in Python models, then SQLMesh needs to use [Databricks Connect](https://docs.databricks.com/dev-tools/databricks-connect.html) to connect to Databricks.
 
-Databricks connect execution can be routed to a different cluster than the SQL Connector by setting the `databricks_connect_*` properties.
-For example this allows SQLMesh to be configured to run SQL on a [Databricks SQL Warehouse](https://docs.databricks.com/sql/admin/create-sql-warehouse.html) while still routing DataFrame operations to a normal Databricks Cluster.
+SQLMesh's Databricks Connect implementation supports Databricks Runtime 13.0 or higher. If SQLMesh detects that you have Databricks Connect installed, then it will use it for all Python models (both Pandas and PySpark DataFrames).
 
-Note: If using Databricks Connect please note the [requirements](https://docs.databricks.com/dev-tools/databricks-connect.html#requirements) and [limitations](https://docs.databricks.com/dev-tools/databricks-connect.html#limitations)
+SQLMesh does not include/bundle the Databricks Connect library. You must [install the version of Databricks Connect](https://docs.databricks.com/en/dev-tools/databricks-connect/python/install.html) that matches the Databricks Runtime used in your warehouse.
```

Comment:
^ This might need some more emphasis. It's non-obvious and definitely tripped me up as a newbie!

Reply:
Agreed - I moved it up top so people can't miss it